test/checks/mobile/target-size.js

```diff
@@ -39,45 +38,139 @@ describe('target-size tests', function () {
+      assert.deepEqual(checkContext._data, {
+        messageKey: 'obscured',
+        minSize: 24,
+        width: 20,
+        height: 20
+      });
+    });
+
+    it('ignores obscuring element has pointer-events:none', function () {
+      var checkArgs = checkSetup(
+        '<a href="#" id="target" style="' +
+          'display: inline-block; width:20px; height:20px;' +
+          '">x</a>' +
+          '<span style="' +
+          'display: inline-block; pointer-events: none; width:20px; height:20px; margin-left: -20px;' +
+          '">x</span>'
+      );
+      assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
+      assert.deepEqual(checkContext._data, {
+        minSize: 24,
+        width: 20,
+        height: 20
+      });
     });
   });
 
-  it('returns false for obscured targets with insufficient space', function () {
-    var checkArgs = checkSetup(
-      '<button id="target" style="' +
-        'display: inline-block; width:40px; height:30px; margin-left:30px;' +
-        '">x</button>' +
-        '<button style="' +
-        'display: inline-block; width:40px; height:30px; margin-left: -10px;' +
-        '">x</button>' +
-        '<button style="' +
-        'display: inline-block; width:40px; height:30px; margin-left: -100px;' +
-        '">x</button>'
-    );
-    assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
-    assert.deepEqual(checkContext._data, {
-      messageKey: 'obscured',
-      minSize: 24,
-      width: 20,
-      height: 30
+  describe('when partially obscured', function () {
+    it('returns true for focusable non-widgets', function () {
+      var checkArgs = checkSetup(
+        '<button id="target" style="' +
+          'display: inline-block; width:40px; height:30px; margin-left:30px;' +
+          '">x</button>' +
+          '<button style="' +
+          'display: inline-block; width:40px; height:30px; margin-left: -10px;' +
+          '">x</button>' +
+          '<span tabindex="0" style="' +
+          'display: inline-block; width:40px; height:30px; margin-left: -100px;' +
+          '">x</span>'
+      );
+      assert.isTrue(check.evaluate.apply(checkContext, checkArgs));
+      assert.deepEqual(checkContext._data, {
+        minSize: 24,
+        width: 30,
+        height: 30
+      });
+    });
+
+    it('returns true for non-focusable widgets', function () {
+      var checkArgs = checkSetup(
+        '<button id="target" style="' +
+          'display: inline-block; width:40px; height:30px; margin-left:30px;' +
+          '">x</button>' +
+          '<button style="' +
+          'display: inline-block; width:40px; height:30px; margin-left: -10px;' +
+          '">x</button>' +
+          '<button disabled style="' +
+          'display: inline-block; width:40px; height:30px; margin-left: -100px;' +
+          '">x</button>'
+      );
+      assert.isTrue(check.evaluate.apply(checkContext, checkArgs));
```

Comment:
I feel this should be `false` as well since the disabled button captures the click on the left side of the target, making the clickable width effectively 20px. 

This can be debatable though as the disabled button doesn't cause a misclick to happen.

<img width="113" alt="image" src="https://user-images.githubusercontent.com/2433219/191381555-f4f50c66-5e8d-4da3-9456-921f1bb8562f.png">

Reply:
Same answer. I don't think we can fail this, maybe incomplete, but I'd prefer we leave that for another time.